src/server/svr_jobfunc.c

```diff
@@ -7329,7 +7298,7 @@ recreate_exec_vnode(job *pjob, char *vnodelist, char *err_msg,
 	free(res_in_exec_vnode);
 	/* clear the summation buffer */
 	(void)resc_sum_values_action(RESC_SUM_GET_CLEAR, NULL, NULL, NULL, buf,
-									sizeof(buf)-1);
+									sizeof(buf));
```

Comment:
If you want to split the above line into two, consider breaking it up more evenly (or just combine it into one line).